model/components/prometheus.yml

```diff
@@ -0,0 +1,105 @@
+---
+components:
+
+  # prometheus
+  - id: prometheus
+    name: Prometheus
+    description: Prometheus is an open-source systems monitoring and alerting toolkit.
```

Comment:
please also a small sentence about exporters, so that it's easier to understand what exporters do.